app/policies/application_policy.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+# Default Policies
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
```

Comment:
If you do `def initialize(user = nil, _record = nil)`, you can get rid of a lot of initializers in other policies (see other comments).